pom.xml

```diff
@@ -234,7 +234,7 @@
       <version.jgroups>4.0.13.Final</version.jgroups>
       <version.jta>1.0.1.Final</version.jta>
       <version.lucene>5.5.5</version.lucene>
-      <version.lucene.module.slot>5.5</version.lucene.module.slot>
+      <version.lucene.module.slot>5.5.5</version.lucene.module.slot>
```

Comment:
This was changed from 5.5.5 to 5.5 in https://issues.jboss.org/browse/ISPN-9410 because of linkage error. Was it causing issues with Widlfly 14?